internal/controller/http/command.go

```diff
@@ -83,12 +83,17 @@ func (c *RestController) SetCommand(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = application.SetCommand(ctx, deviceName, commandName, queryParams, requestParamsMap, c.dic)
+	event, err := application.SetCommand(ctx, deviceName, commandName, queryParams, requestParamsMap, c.dic)
```

Comment:
modify the swagger file for this API enhanchement as well

Reply:
add a statment in the description
https://github.com/edgexfoundry/device-sdk-go/blob/84b3eaa6b9d9b6bc9df706aeb119f986429132ec/openapi/v3/device-sdk.yaml#L437

Reply:
since the docs PR is opened, resolving this change request